Realm/RLMProperty.mm

```diff
@@ -39,7 +39,6 @@ - (instancetype)initWithName:(NSString *)name
         _objectClassName = objectClassName;
         _indexed = indexed;
         [self setObjcCodeFromType];
-        [self updateAccessors];
```

Comment:
That change can be split out of this PR if desired; it was just a perf optimization to try to make up for the added overhead of opening the named pipe.